packages/indy-sdk/src/anoncreds/services/IndySdkAnonCredsRegistry.ts

```diff
@@ -509,6 +511,47 @@ export class IndySdkAnonCredsRegistry implements AnonCredsRegistry {
       }
     }
   }
+
+  private async fetchIndySchemaWithSeqNo(agentContext: AgentContext, seqNo: number, did: string) {
+    const indySdkPoolService = agentContext.dependencyManager.resolve(IndySdkPoolService)
+    const indySdk = agentContext.dependencyManager.resolve<IndySdk>(IndySdkSymbol)
+
+    const { pool } = await indySdkPoolService.getPoolForDid(agentContext, did)
+    agentContext.config.logger.debug(`Getting transaction with seqNo '${seqNo}' from ledger '${pool.didIndyNamespace}'`)
+
+    const request = await indySdk.buildGetTxnRequest(did, 'DOMAIN', seqNo)
+
+    agentContext.config.logger.trace(`Submitting get transaction request to ledger '${pool.didIndyNamespace}'`)
+    const response = await indySdkPoolService.submitReadRequest(pool, request)
+
+    const schema = response.result.data as SchemaType
```

Comment:
We should do the same check here as we do with the `IndyVdrAnonCredsRegistry` and check if the txn type is `101`